src/view/com/composer/photos/ImageAltTextDialog.tsx

```diff
@@ -21,32 +25,48 @@ type Props = {
   Portal: PortalComponent
 }
 
-export const ImageAltTextDialog = (props: Props): React.ReactNode => {
+export const ImageAltTextDialog = ({
+  control,
+  image,
+  onChange,
+  Portal,
+}: Props): React.ReactNode => {
+  const altTextRef = React.useRef<string>(image.alt)
```

Comment:
hm, another ref. i’m worrying about proliferation of refs — why are we adding them? they should only be used as escape hatches and i don’t yet see what we’re escaping from. generally adding them for normal data flow needs to have some kind of explanation so we don’t miss this in reviews.